build fail looks real:
```
======================================================================
ERROR: test_form_workflow_module (corehq.apps.app_manager.tests.test_suite.SuiteTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_suite.py", line 308, in test_form_workflow_module
    self.assertXmlPartialEqual(self.get_xml('suite-workflow-module'), app.create_suite(), "./entry")
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/models.py", line 3570, in create_suite
    return suite_xml.SuiteGenerator(self).generate_suite()
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 680, in generate_suite
    self.post_process(suite)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 742, in post_process
    self.add_form_workflow(suite)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 888, in add_form_workflow
    create_workflow_stack(suite, form_command, frame_children)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/suite_xml.py", line 789, in create_workflow_stack
    frame = CreateFrame(if_clause=unescape(if_clause))
  File "/usr/lib/python2.7/xml/sax/saxutils.py", line 53, in unescape
    data = data.replace("&lt;", "<")
AttributeError: 'NoneType' object has no attribute 'replace'
----------------------------------------------------------------------
```

--- a/corehq/apps/app_manager/suite_xml.py
+++ b/corehq/apps/app_manager/suite_xml.py
@@ -9,7 +9,7 @@ import urllib
 
 from eulxml.xmlmap import StringField, XmlObject, IntegerField, NodeListField, NodeField, load_xmlobject_from_string
 from lxml import etree
-from xml.sax.saxutils import escape
+from xml.sax.saxutils import escape, unescape
 
 from django.core.urlresolvers import reverse
 
@@ -786,6 +786,7 @@ class SuiteGenerator(SuiteGeneratorBase):
                 template = '({{}}) and ({})'.format(if_clause) if if_clause else '{}'
                 if_clause = template.format(if_prefix)
 
+            if_clause = unescape(if_clause) if if_clause else None
             frame = CreateFrame(if_clause=if_clause)
             entry.stack.add_frame(frame)
 

--- a/corehq/apps/app_manager/tests/data/suite/form_link_basic.xml
+++ b/corehq/apps/app_manager/tests/data/suite/form_link_basic.xml
@@ -6,7 +6,7 @@
             </text>
         </command>
         <stack>
-          <create if="true()">
+          <create if="(today() - dob) &lt; 7">
             <command value="'m1'"/>
             <command value="'m1-f0'"/>
           </create>

--- a/corehq/apps/app_manager/tests/data/suite/form_link_update_case.xml
+++ b/corehq/apps/app_manager/tests/data/suite/form_link_update_case.xml
@@ -11,7 +11,7 @@
             <datum id="case_id" nodeset="instance('casedb')/casedb/case[@case_type='frog'][@status='open']" value="./@case_id" detail-select="m0_case_short" detail-confirm="m0_case_long"/>
         </session>
         <stack>
-          <create if="true()">
+          <create if="(today() - dob) &gt; 7">
             <command value="'m1'"/>
             <datum id="case_id" value="instance('commcaresession')/session/data/case_id"/>
             <command value="'m1-f0'"/>

--- a/corehq/apps/app_manager/tests/test_suite.py
+++ b/corehq/apps/app_manager/tests/test_suite.py
@@ -838,7 +838,7 @@ class TestFormLinking(SimpleTestCase, TestFileMixin):
 
         m0f0.post_form_workflow = WORKFLOW_FORM
         m0f0.form_links = [
-            FormLink(xpath='true()', form_id=m1f0.unique_id)
+            FormLink(xpath="(today() - dob) &lt; 7", form_id=m1f0.unique_id)
         ]
         self.assertXmlPartialEqual(self.get_xml('form_link_basic'), app.create_suite(), "./entry[1]")
 
@@ -852,7 +852,7 @@ class TestFormLinking(SimpleTestCase, TestFileMixin):
 
         m0f0.post_form_workflow = WORKFLOW_FORM
         m0f0.form_links = [
-            FormLink(xpath='true()', form_id=m1f0.unique_id)
+            FormLink(xpath="(today() - dob) > 7", form_id=m1f0.unique_id)
         ]
 
         self.assertXmlPartialEqual(self.get_xml('form_link_update_case'), app.create_suite(), "./entry[1]")

